terraspace/app/modules/cluster-autoscaler/variables.tf

```diff
@@ -0,0 +1,21 @@
+variable "service_account_name" {
+  type    = string
+  default = "cluster-autoscaler"
+}
+
+variable "namespace" {
+  type    = string
+  default = "kube-system"
+}
+
+variable "cluster_name" {
+  type = string
```

Comment:
remind me, we should look at including tflint on module changes, so that we cover items like adding `description` on variables.

Reply:
Yes that makes sense. I just wouldn't block this specific PR over that. Let's add this for next ones?

Reply:
Correct.  No need to block this PR due to that.